webserver/views/api/v1/dataset_eval.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+from webserver.decorators import auth_required
+import db.dataset_eval
+
+bp_dataset_eval = Blueprint('api_v1_dataset_eval', __name__)
+
+@bp_dataset_eval.route("/pending-jobs", methods=["GET"])
+@auth_required
+def pending_jobs():
+    """Return a list of pending jobs for a user to be evaluated remotely.
+       This method is valid only for Authenticated users."""  
```

Comment:
This part still needs to have better documentation. Provide information about return values. See https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/views/api/v1/datasets.py#L25-L67 for example.